content/en/docs/tutorials/pod-security/cluster-level-pss.md

```diff
@@ -0,0 +1,271 @@
+        extraVolumes:
+          - name: accf
+            hostPath: /etc/config
+            mountPath: /etc/config
+            readOnly: false
+            pathType: "DirectoryOrCreate"
+  extraMounts:
+  - hostPath: /tmp/pss
+    containerPath: /etc/config
+    # optional: if set, the mount is read-only.
+    # default false
+    readOnly: false
+    # optional: if set, the mount needs SELinux relabeling.
+    # default false
+    selinuxRelabel: false
+    # optional: set propagation mode (None, HostToContainer or Bidirectional)
+    # see https://kubernetes.io/docs/concepts/storage/volumes/#mount-propagation
+    # default None
+    propagation: None
+EOF
+```
+
+_Note_: Please ensure that when using Docker Desktop with KinD, the `/tmp`
+directory is added as a Shared Directory under
+`Preferences -> Resources -> File Sharing` (as seen on Docker Desktop for Mac)
+
+# Create cluster with Pod Security Standards
+
+Now we are ready to create a cluster that consumes these pod security standards
+configured with pod security admission controller:
+
+```
+$ kind create cluster --name psa-with-cluster-pss --image kindest/node:latest --config /tmp/pss/cluster-config.yaml
+Creating cluster "psa-with-cluster-pss" ...
+ ✓ Ensuring node image (kindest/node:latest) 🖼 
+ ✓ Preparing nodes 📦  
+ ✓ Writing configuration 📜 
+ ✓ Starting control-plane 🕹️ 
+ ✓ Installing CNI 🔌 
+ ✓ Installing StorageClass 💾 
+Set kubectl context to "kind-psa-with-cluster-pss"
+You can now use your cluster with:
+
+kubectl cluster-info --context kind-psa-with-cluster-pss
+
+Have a question, bug, or feature request? Let us know! https://kind.sigs.k8s.io/#community 🙂
+
+
+$ kubectl cluster-info --context kind-psa-with-cluster-pss
+Kubernetes control plane is running at https://127.0.0.1:63855
+CoreDNS is running at https://127.0.0.1:63855/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+```
+
+# Create pod
+
+Creating a pod with minimal configuration in default namespace should now be
+successful with warnings for `restricted` pod security standards:
+
+```
+$ cat <<EOF > /tmp/pss/nginx-pod.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+    - image: nginx
+      name: nginx
+      ports:
+        - containerPort: 80
+EOF
+
+
+$ kubectl apply -f /tmp/pss/nginx-pod.yaml
+Warning: would violate PodSecurity "restricted:latest": allowPrivilegeEscalation != false (container "nginx" must set securityContext.allowPrivilegeEscalation=false), unrestricted capabilities (container "nginx" must set securityContext.capabilities.drop=["ALL"]), runAsNonRoot != true (pod or container "nginx" must set securityContext.runAsNonRoot=true), seccompProfile (pod or container "nginx" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")
+pod/nginx created
+```
+
+# Bringing it all together
```

Comment:
I think you should move this to "What's Next" and say "Run a gist to perform all the preceding steps at once"